### PR TITLE
adding recovery integration tests

### PR DESCRIPTION
**Pull request by acharb**
[ticket](https://stellarorg.atlassian.net/browse/WAL-572?atlOrigin=eyJpIjoiNWVjYzAzZmQ2NmNiNDM5M2FmYjg5ODU2ODM2ZWQ4YzgiLCJwIjoiaiJ9)

integration tests for testing the Recovery code

### CODE REVIEW

File: test/docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+version: "3"
```
Comment (Ifropc) [inline test/docker/docker-compose.yml:1]: Can you please fix versions on images in this file?

File: test/docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+services:
```
Comment (Ifropc) [inline test/docker/docker-compose.yml:2]: Do you mind adding a short comment or maybe even a small README about integration test setup and how do they work?

**Review by Ifropc — APPROVED** [review]

File: test/docker/docker-compose.yml (hunk 2 above)
Comment (acharb) [reply test/docker/docker-compose.yml:2]: ya good idea https://github.com/stellar/typescript-wallet-sdk/pull/74/commits/706c890ed3a1b5899d1a5ecdee3892c04930f664

File: test/docker/docker-compose.yml (hunk 1 above)
Comment (acharb) [reply test/docker/docker-compose.yml:1]: https://github.com/stellar/typescript-wallet-sdk/pull/74/commits/706c890ed3a1b5899d1a5ecdee3892c04930f664

**Review by CassioMG — APPROVED** [review]
Nice!

**Pull request merged**